src/configuration/configuration.js

```diff
@@ -1 +1 @@
-export const configuration = require ('./configuration.json');
+export default require ('./configuration.json');
```

Comment:
Let's sync up about module syntax and exports/imports and figure out how we want to do this one. There may be more to this than simply switching it to be `default` instead of named.

Reply:
I would like to talk module export/import syntax  as well.  I don't understand the tradeoffs of the different options and what is best practice.